src/Day.jsx

```diff
@@ -7,6 +7,14 @@ let Day = React.createClass({
 
   propTypes: TimeGrid.propTypes,
 
+  getDefaultProps() {
+    return {
+      now: new Date(),
+      type: 'gutter',
+      timeslots: 2
+    }
```

Comment:
I would like to go with the 2nd choice.  The reason is that even though you don't wish to document the internals so we don't run into backwards compatibility hell, if anyone does need to make a change, it's much harder to do it unless all the needed props are documented and required ones explicitly marked as so.  This is why I always mark props that are absolutely needed for the component to function as required, even if they have a default value supplied.  It's redundant, but this stuff is only checked in development anyways, so I think of it as another layer of documentation for the developer.  So is it cool to use the default props for that reason?